syntax = "proto3";

import "Task.proto";

package model;

message TaskNode {
  Task task = 1;
  TaskId parent_id = 2;
}